consensus/src/context/rx_vms.rs

```diff
@@ -1,6 +1,6 @@
-//! RandomX VM Cache
+//! `RandomX` VM Cache
 //!
-//! This module keeps track of the RandomX VM to calculate the next blocks PoW, if the block needs a randomX VM and potentially
+//! This module keeps track of the `RandomX` VM to calculate the next blocks `PoW`, if the block needs a randomX VM and potentially
```

Comment:
Should PoW be in ``?

Reply:
No but unfortunately it triggers the CamelCase doc lint, changed to proof-of-work.